tests/test_endpoints.py

```diff
@@ -124,8 +120,8 @@ def test_search(client, page_id):
 
 
 @pytest.mark.vcr()
-def test_databases_create(client, page_id):
-    db_name = f"Test Database - {datetime.now()}"
+def test_databases_create(client, page_id, test_page_name):
+    db_name = f"Test Database - {test_page_name}"
```

Comment:
Why is it better that way?

Reply:
Just for convenience I removed the datetime module and just used the fixture. Totally an aesthetic choice, I didn't like to use the import just for that single line

Reply:
I'm not fond of the final name ("Test Database - Test Page - date"). We can simply remove the datetime logic entirely and call the database like the test, i.e. `test_databases_create`. The page that we use as a parent (with `page_id`) already has a unique name based on the datetime so no clash would be possible if we use the test name. What do you think?

Reply:
I agree with you, the parent page should be enough. Just calling them Test Database then :)